app/controllers/v1/interventions/predefined_participants_controller.rb

```diff
@@ -37,10 +37,20 @@ def update
     render json: serialized_response(predefined_user)
   end
 
+  def destroy
+    authorize! :read, Intervention
+    authorize! :read, intervention_load
```

Comment:
I think that the user should at least have access to the update to perform this action

Reply:
Also, look at https://github.com/Michigan-State-University/cias-api/pull/126/files